Gray Code/Gray Code.cpp

```diff
@@ -0,0 +1,53 @@
+#include<bits/stdc++.h>
```

Comment:
File name should be  Gray_Code/Gray_Code.cpp

Reply:
_ is included.

Reply:
done @somya-kapoor